api/meter.go

```diff
@@ -163,18 +187,24 @@ func (r *relayMeter) loadData(from, to time.Time) error {
 
 	if updateDaily {
 		r.dailyUsage = dailyUsage
+
```

Comment:
Please remove this extra blank line introduced by the PR.

Reply:
Same as below.